packages/time-picker/src/time-picker.ts

```diff
@@ -32,15 +32,22 @@ export default defineComponent({
 
     provide('ElPopperOptions', props.popperOptions)
     ctx.expose(refProps)
-    return () => h(Picker, {
-      format: DEFAULT_FORMATS_TIME,
-      ...props, // allow format to be overwrite
-      type,
-      ref: commonPicker,
-      'onUpdate:modelValue': value => ctx.emit('update:modelValue', value),
-    },
-    {
-      default: scopedProps => h(panel, scopedProps),
-    })
+    console.log(props)
```

Comment:
:)

Reply:
Nope. This was my bad. I will add an plugin to remove `console.log` statement to make sure this won't happen again

Reply:
Just got it fixed. Upgrade to beta48 then you won't see those annoying logs.